version.props

```diff
@@ -1,6 +1,10 @@
-<!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <ExperimentalProjectVersionPrefix>0.3.1</ExperimentalProjectVersionPrefix>
```

Comment:
just FYI - I've found it's too easy to forget to update a VersionPrefix set in csproj directly. I'm hoping putting all versions in version.props will help us when we have multiple version prefixes coming from one repo.

Reply:
Ah that's why you moved it here. 👍